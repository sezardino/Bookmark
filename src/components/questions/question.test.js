import React from 'react';
import {shallow} from 'enzyme';

import Question from './question';

describe('Component Question', () => {
	const mockData = {question: 'question', answer: 'answer', id: '1'};
	const component = shallow(<Question data={mockData} />);

	it('Component Question rendered correctly', () => {
		expect(component.debug()).toMatchSnapshot();
	});
});
