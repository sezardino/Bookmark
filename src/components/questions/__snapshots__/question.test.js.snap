// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Question Component Question rendered correctly 1`] = `
"<li className=\\"questions__item \\">
  <button className=\\"button button--questions questions__button\\" onClick={[Function: onClick]}>
    question
    <span className=\\"button button--arrow \\">
      <svg xmlns=\\"http://www.w3.org/2000/svg\\" width={18} height={12}>
        <path className=\\"arrow\\" fill=\\"none\\" stroke=\\"#5267DF\\" strokeWidth={3} d=\\"M1 1l8 8 8-8\\" />
      </svg>
    </span>
  </button>
  <p className=\\"questions__text\\">
    answer
  </p>
</li>"
`;
